scripts/design-system/agent-eval/run.mjs

```
@@ -134,4 +134,27 @@ if (!buttonEntry?.agent?.variants?.variant?.values?.length) {
   console.log("✓ Button agent variants extracted from source");
 }
 
+const goldenPath = join(dirname(fileURLToPath(import.meta.url)), "golden-intents.json");
```

ESLint

[error] Replace `dirname(fileURLToPath(import.meta.url)),·"golden-intents.json"` with `⏎··dirname(fileURLToPath(import.meta.url)),⏎··"golden-intents.json",⏎`

(prettier/prettier)